strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/registry/support/ExternalAuthenticatorsHelper.java

```diff
@@ -17,20 +17,20 @@
 /**
  * @author Przemyslaw Fusik
  */
-public class AuthenticatorsClassLoader
+public class ExternalAuthenticatorsHelper
```

Comment:
Regarding [this](https://github.com/strongbox/strongbox/pull/278/files#diff-5007dae3de712baa3e32e2b397248eccL40), `WEB-INF/lib` is already in the `classpath`, so I suppose that we don't need this default value.
In general, I can't say that I like this reflection with `"addURL"` to add `Authenticator` jar files. I suggest you to use common `URLClassLoader` for this like below:
```
        ClassLoader currentClassLoader = Thread.currentThread().getContextClassLoader();
        URLClassLoader urlClassLoader = new URLClassLoader("path/to/authenticator/jar/files", currentClassLoader);
        Thread.currentThread().setContextClassLoader(urlClassLoader);
        //Load everything you need here
        Thread.currentThread().setContextClassLoader(currentClassLoader);
``` 
This looks clearly and will work independently of the environment (WAS, WebLogic. Jetty, Tomcat etc.).